config/rdbsub.csv

```diff
@@ -0,0 +1,3 @@
+tabname,filts,columns
+trade,sym=`GOOG,
+quote,bid>100.0,
```

Comment:
This file will need removed before release
Fine for something like this to live in FSP